src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -268,6 +268,11 @@ final class SecureStore: SecureKeyValueStoring, Store, AntigenTestProfileStoring
 		get { kvStore["dccAdmissionCheckScenarios"] as DCCAdmissionCheckScenarios? ?? nil }
 		set { kvStore["dccAdmissionCheckScenarios"] = newValue }
 	}
+
+	var regroupingAfterMigrationChanged: Bool {
```

Comment:
I'm a bit unsure about the name, either I would go with something like "certificateGroupingChanged" or "migrationChangedCertificateGrouping" or to stay closer to you original "groupingChangedAfterMigration". Or it could go the route of something like "shouldShowRegroupingHint" or "shouldShowRegroupingAlert", then the reset to false after showing it feels a bit cleaner 🤓

Reply:
Yeah, I like the idea naming it to `shouldShowRegroupingAlert` because it describes the best for what the property is for. I'll change it.